event-session-signup/README.md

```diff
@@ -0,0 +1,13 @@
+# Sign Up for Sessions at an Event
+Allows event attendees to sign up for sessions at an event and creates an itinerary.
+
+## Description
+Quickly create an end to end conference registration system. Start with the conference event information listed in a Google Sheet, then use Apps Script to create a sign-up form and a personalized itinerary for each attendee. For each attendee, a personalized schedule in a Google Doc is emailed out AND a conference calendar is created then populated with the specific sessions.
+
+## Set Up Instructions
+1. Create a copy of the sample [Event Session spreadsheet](https://docs.google.com/spreadsheets/d/1cpGsysprd5zl8VHYs4njsTSXwGxMA7DZPCW5o2EhR_8/copy)
+1. Click on custom menu item 'Conference' > 'Set up conference'
+1. A dialog box will appear and tell you that the script requires authorization. Read the authorization notice and 1. continue.
```

Comment:
re: "and 1. continue.", is the "1." a typo?

Reply:
yes, thank you for noticing!